Microsoft.Xbox.Service.DevTools/XblConfig/Product.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Xbox.Services.DevTools.XblConfig
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Represents a product within Dev Center.
+    /// </summary>
+    public class Product
+    {
+        /// <summary>
+        /// Gets or sets the account ID associated with this product.
+        /// </summary>
+        public Guid AccountId { get; set; }
+
+        /// <summary>
+        /// Gets or sets any alternate IDs for this product.
+        /// </summary>
+        public IEnumerable<AlternateId> AlternateIds { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating if this product is a test product or not.
+        /// </summary>
+        public bool IsTest { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Microsoft Account application ID.
+        /// </summary>
+        public string MsaAppId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the package family name ID.
+        /// </summary>
+        public string PfnId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the primary service configuration ID.
+        /// </summary>
+        public Guid PrimaryServiceConfigId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the product ID.
+        /// </summary>
+        public Guid ProductId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the title ID.
+        /// </summary>
+        public uint TitleId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Xbox Live tier.
+        /// </summary>
+        public XboxLiveTier XboxLiveTier { get; set; }
+
+        /// <inheritdoc/>
+        public override string ToString()

```

Comment:
>ToString [](start = 31, length = 8)

what is this for?

Reply:
For a string representation of a product.

---
In reply to: [175227831](https://github.com/Microsoft/xbox-live-developer-tools/pull/37#discussion_r175227831) [](ancestors = 175227831)